test-cases/longevity/longevity-many-clients-4h.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 360
+
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=240m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+             "cassandra-stress read  cl=QUORUM duration=240m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
```

Comment:
I think you'll need to reduce the amount of threads.
If you have 50 client each one will run 2  commands with 40 threads each, the load may be too high.

Reply:
Reduced. This was taken from longevity 4h. I did some math now, and assuming on longevity-4h was around 1.67 stress thread per scylla node core (2 loaders x 80 threads / 96 cores = 1,67 threads/core), I kept the similar ratio for this test resulting in 7 thread (50 loaders x 7 threads / 216 cores = 1,62 threads/core). Let me know if this thinking is ok.